README.md

```diff
@@ -111,6 +111,15 @@ Si deseas agregar algun canal o tienes alguna sugerencia, puedes usar [@openve](
 * [Un Python Al Día](https://t.me/UnPythonAlDia) - @kotov52
 * [Wikimedia Venezuela](https://t.me/wikimediave) - @oscar_costero - @Dennis6492
 
+### Canales de noticias
+
+* [RDN Digital](https://t.me/RDNDigital)
+* [Caraota Digital](https://t.me/elgerminadorcaraotadigital)
+* [Efecto Cocuyo](https://t.me/efectococuyo)
+* [Tal Cual](https://t.me/TalCualDigital)
+* [Genbeta](https://t.me/genbeta)
+* [Xakata](https://t.me/xakata)
```

Comment:
Creo que este canal sale del contexto del repo y de OpenVe.

Reply:
@Villanuevand @foursixnine las agregué porque noté que se ha extendido el contexto del repo a mi parecer. Pero ya queda en la decisión de ustedes.

Reply:
@foursixnine  exactamente por eso:  recordando la premisa de la organización: 
>  OpenVE es una comunidad libre para la investigación y el desarrollo de proyectos de computación de código abierto.